src/core/lib/gprpp/arena.h

```diff
@@ -77,6 +84,9 @@ class Arena {
   };
 
   explicit Arena(size_t initial_size) : initial_zone_size_(initial_size) {}
```

Comment:
1) this should call Arena(initial_size, 0). 
2) I'd just make it one ctor `explicit Arena(size_t initial_size, size_t initial_alloc=0)`.

Reply:
Done